src/algorithms/calorimetry/CalorimeterClusterRecoCoG.cc

```diff
@@ -6,6 +6,7 @@
  *  Logarithmic weighting is used for mimicing energy deposit in transverse direction
  *
  *  Author: Chao Peng (ANL), 09/27/2020
+            Dhevan Gangadharan (UH): cluster profiling from Eigenvalues
```

Comment:
Please don't keep revision story in comments. Add yourself to the SPDX copyright section above instead. By doing that you confirm that you are the sole author of the changes and have rights to issue your code under LGPL3 or newer.

Reply:
Done.